packages/manager/apps/telecom/src/components/notification/notification-list.html

```diff
@@ -3,6 +3,33 @@
     <h2 data-translate="components_notification_loading"></h2>
 </div>
 
+<div data-ng-hide="NotificationListCtrl.loading">
```

Comment:
```suggestion
<div data-ng-if="!NotificationListCtrl.loading">
```
if possible, that way element is not present in the DOM if not required to (instead of just not being displayed)